.github/workflows/doc-build.yml

```
@@ -95,10 +95,11 @@ jobs:
         # If matches a tag like refs/tags/v1.12.0-rc3 or
         # refs/tags/v1.12.0 convert to 1.12
         GITHUB_REF=${{ github.ref }}
+        export GITHUB_REF=${{ github.ref }}
 
         # Convert refs/tags/v1.12.0rc3 into 1.12.
         # Adopted from https://github.com/pytorch/pytorch/blob/main/.github/workflows/_docs.yml#L150C11-L155C13
-        if [[ "${GITHUB_REF}" =~ ^refs/tags/v([0-9]+\\.[0-9]+)\\. ]]; then
+        if [[ "${GITHUB_REF}" =~ ^refs/tags/v([0-9]+\.[0-9]+)\.* ]]; then
```

Could simplify this to
```suggestion
        if [[ "${GITHUB_REF}" =~ ^refs/tags/v([0-9]+\.[0-9]+) ]]; then
```
The trailing `\.*` says "match zero or more dot characters" rather than "match zero or more of any character". But since we're just looking at the prefix (there's no `$`), we can just describe the prefix.

Tested with
```
GITHUB_REF=refs/tags/v0.2.0-rc5
if [[ "${GITHUB_REF}" =~ ^refs/tags/v([0-9]+\.[0-9]+) ]]; then
  echo "${GITHUB_REF} is a release tag"
else
  echo "${GITHUB_REF} is NOT a release tag"
fi
```
```
% bash -x /tmp/test.sh
+ GITHUB_REF=refs/tags/v0.2.0-rc5
+ [[ refs/tags/v0.2.0-rc5 =~ ^refs/tags/v([0-9]+\.[0-9]+) ]]
+ echo 'refs/tags/v0.2.0-rc5 is a release tag'
refs/tags/v0.2.0-rc5 is a release tag
```